### PR TITLE
Feature/twitter/image tweet

### PR DESCRIPTION
**Pull request by J0N4S13**
## Description
Image upload code in twitter commented, in the future send a gif instead of a png.

## Task items:
- [ ] Image upload code on twitter commented;

### CODE REVIEW

File: src/services/Warnings.js [hunk 1]
```diff
@@ -6,7 +6,7 @@
  */
 
 const moment = require('moment');
-const fs = require('fs');
+// const fs = require('fs');
```
Comment (quetzyg) [inline src/services/Warnings.js:9]: If the code isn't required, just remove it.

File: src/services/Warnings.js [hunk 2]
```diff
@@ -176,11 +176,13 @@ const getWarningsZones = (warningsZone, zone, client) => {
 
     // Send message to Twitter
     if (clientTwitter && resptwitter !== '') {
-      if (level === 'Amarelo') {
+      // Review the code to send GIF
```
Comment (quetzyg) [inline src/services/Warnings.js:179]: Same as above, commented code shouldn't be committed.

File: src/services/Warnings.js [hunk 3]
```diff
@@ -176,11 +176,13 @@ const getWarningsZones = (warningsZone, zone, client) => {
...
+      /* if (level === 'Amarelo') {
         // Load your image
-        const data = fs.readFileSync('/images/VOSTPT_YellowWarning.png');
+        const data = fs.readFileSync('../images/VOSTPT_YellowWarning.png');
         // Make post request on media endpoint. Pass file data as media parameter
         clientTwitter.post('media/upload', { media: data }, (error, media, response) => {
+          console.log(error);
```
Comment (quetzyg) [inline src/services/Warnings.js:185]: Also, avoid the use of `console.log()`

**Review by quetzyg — CHANGES_REQUESTED** [review]
Left some comments.

**Comment by miguelcarreiro** [comment]
I think that this PR should be closed, since this feature was implemented in PR https://github.com/vostpt/bot/pull/72

**Pull request closed without merging**